frontend/src/components/GoesData/LiveTab.tsx

```
@@ -634,7 +632,12 @@ export default function LiveTab({ onMonitorChange }: Readonly<LiveTabProps> = {}
         )}
 
         {/* Top controls overlay — on mobile, hidden unless overlayVisible */}
-        <div className={`absolute top-0 inset-x-0 z-10 bg-gradient-to-b from-black/70 via-black/30 to-transparent pointer-events-none transition-opacity duration-300 ${isMobile && !overlayVisible ? 'opacity-0 pointer-events-none' : 'opacity-100'}`} data-testid="controls-overlay">
+        <div
+          className={`absolute top-0 inset-x-0 z-10 bg-gradient-to-b from-black/70 via-black/30 to-transparent pointer-events-none transition-opacity duration-300 ${!overlayVisible ? 'opacity-0 pointer-events-none' : 'opacity-100'}`}
+          data-testid="controls-overlay"
+          onMouseEnter={() => { setOverlayVisible(true); clearTimeout(overlayTimer.current); }}
+          onMouseLeave={resetOverlayTimer}
+        >
```

Biome (2.4.4)

[error] Static Elements should not be interactive.

(lint/a11y/noStaticElementInteractions)